test/commonASTtests/tester.py

```diff
@@ -0,0 +1,77 @@
+import sys
+import subprocess
+import glob
+import os
+
+print "Python files"
+print
+
+for fname in glob.glob("py-test-files/*.py"):
+	print
+	print "RUNNING RUNNER ON ", fname
+	subprocess.call(["python", "runner.py", "-py", fname])
+
+print
+print "C++ files"
+print
+
+for fname in glob.glob("/mnt/c/Users/Elizabeth/Documents/spring2017/commonASTs/cpp-test-files/*.cpp"):
```

Comment:
add whatever files you want to test against to the repository and change the path to use them as right now this tester is useless for anyone but you.